app/src/scripts/dataset/dataset.metrics.jsx

```diff
@@ -22,12 +22,17 @@ class Metrics extends React.Component {
     let displayFollowers = fromDashboard
       ? followers !== '0' ? true : false
       : true
-
+    let displayViews = fromDashboard ? false : true
```

Comment:
views should not be displayed when the project is a draft - see displayDownloads above